spec/services/manifest_builder_v3/by_resource_type/manifest_builder_v3_playlist_spec.rb

```diff
@@ -79,9 +84,6 @@
       it "generates the Canvases for the FileSets" do
         expect(output).not_to be_empty
 
-        expect(output).to include("rendering")
-        expect(output["rendering"]).to be_empty
```

Comment:
How do we provide access to a canvas' full tiff download? Is the rendering somewhere else? Or do we still have canvas renderings and this just makes sure we don't have blank values?